cypress/jenkins/init.sh

```diff
@@ -59,6 +59,8 @@ if [ -f "${PRIV_KEY}" ]; then rm "${PRIV_KEY}"; fi
 ssh-keygen -t ecdsa -b 521 -N "" -f "${PRIV_KEY}"
 ls -al "${WORKSPACE}/.ssh/"
 
+cat "${PRIV_KEY}"
```

Comment:
is this debug? feels odd outputting a private key to logs

Reply:
Hi Richard thanks for taking a look to this PR.

This is an ephemeral key that lasts until the node that execute the tests is deleted by corral in the Jenkins pipeline. Printing the key in the pipeline console output is helpful for debugging issues in the node in case we decide to run the pipeline with `CLEANUP=false`.

I can remove it tho if we only want to have this output during local runs.

Reply:
Just on the paranoid side i guess, could you gate it on that `CLEANUP` flag?

Reply:
I gave it another thought and the branch can be configured in the pipeline if there's need for debugging. I'll just remove it as you have a good point.